test/Spaces/opt_spaces.jl

```diff
@@ -35,7 +35,7 @@ end
 #! format: off
     if ClimaComms.device(context) isa ClimaComms.CUDADevice
         test_n_failures(91,   TU.PointSpace, context)
-        test_n_failures(141,  TU.SpectralElementSpace1D, context)
+        test_n_failures(825,  TU.SpectralElementSpace1D, context)
```

Comment:
Can you please look and see what this spike is from?

Reply:
We might be able to use a function barrier to substantially reduce it

Reply:
This spike is caused by the conversion of `Array`s to `CuArray`s, and it is similar to the spike in JET failures from CPUs to GPUs for `SpectralElementSpace2D`.

Reply:
If we need to reduce this for `SpectralElementSpace1D`, we should probably apply the same fix to `SpectralElementSpace2D`

Reply:
It turns out that these are all false warnings, induced by our `get!` caching. Commenting out `get!` in spectral element grids and interval meshes shows no inference failures.